libs/stats/odc/stats/tasks.py

```diff
@@ -55,16 +58,35 @@ def sanitize(v):
         if isinstance(v, datetime):
             return v.isoformat()
         return v
+
     return transform_object_tree(sanitize, query)
 
 
+def download_from_s3(cache):
```

Comment:
Must support output to specified location, only default to object name if user hasn't supplied desired output location

Reply:
Added support for this.